web-ui/src/api/roles.js

```diff
@@ -19,6 +19,14 @@ export const getAllUserRoles = async (cookie) => {
   });
 };
 
+export const getCurrentUserRole = async (memberId, cookie) => {
+  return resolve({
+    url: `${roleURL}/${memberId}`,
+    responseType: "json",
+    headers: { "X-CSRF-Header": cookie },
+  });
+};
+
```

Comment:
This function is not being used anywhere

Reply:
Gone.